### ISSUE TITLE
[GroupedList][Accessibility] - Any status role inside GroupedList is not read by narrator on Edge.

### ISSUE THREAD

**Issue by salehaHussain** [OfficeDev/office-ui-fabric-react#3699]
### Bug Report

- __Package version(s)__: Latest
- __Browser and OS versions__: Edge Narrator Combo, Windows 10

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: High

Products/sites affected: VS Marketplace

#### Describe the issue:

Having an aria role status inside a role="listitem" parent, will prevent Narrator from reading it aloud. Thus all Spinners and Message Bars will not be accessible inside a list.

#### Actual behavior:

The status is not read out loud.

#### Expected behavior:

The status should be read out loud, as it is in NVDA.

### If applicable, please provide a codepen repro:

https://codepen.io/salehahu/pen/dJmXEM

**Comment by michaelangotti** [comment]
@cschlechty Can you take a look at this issue and comment?

**Comment by JasonGore** [comment]
@salehaHussain Is this still an issue? I checked your CodePen but I think its contents may have changed since you posted this as it seems to be a controlled CheckBox example. Thanks.

**Comment by code-mage** [comment]
This is definitely still an issue.

**Comment by JasonGore** [comment]
@code-mage Please open a new issue (ideally with CodePen repro.) Thanks!
